grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -256,8 +256,8 @@ class ShipmentItem implements Comparable, Serializable {
         return inventoryItem?.lotStatus == LotStatusCode.RECALLED
     }
 
-    int getQuantityInvoiced() {
-        return invoiceItems.sum { it.quantity ?: 0 }
+    Integer getQuantityInvoiced() {
```

Comment:
out of curiosity - was there a reason for that change? `invoiceItems` seems to be a collection, so it should never be `null` - in the "worst case scenario" it should be an empty Set.

Reply:
When I was trying to evaluate my code from the validating method I saw it was throwing an exception (as I remember, it was in case when I rolled back stock movement and deleted the connected invoice)